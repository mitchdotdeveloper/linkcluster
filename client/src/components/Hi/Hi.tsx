export type Message = {
  message: string;
};

export const Hi = ({ message }: Message) => <h1>{`Hi! ${message}`}</h1>;
