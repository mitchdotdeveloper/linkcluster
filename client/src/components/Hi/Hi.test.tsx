import { Hi } from 'components/Hi/Hi';
import { render } from '@testing-library/react';

describe('test', () => {
  it('renders', () => {
    const { asFragment } = render(<Hi message={'My name is Mitch!'} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
