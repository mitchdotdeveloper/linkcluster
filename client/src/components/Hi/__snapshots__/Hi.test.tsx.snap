// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test renders 1`] = `
<DocumentFragment>
  <h1>
    Hi! My name is Mitch!
  </h1>
</DocumentFragment>
`;
